src/components/FormInput/FormInput.test.tsx

```diff
@@ -0,0 +1,88 @@
+import userEvent from '@testing-library/user-event';
+import { useState } from 'react';
+import { describe, it, expect, vi } from 'vitest';
+
+import { render, screen } from '../../test/test-utils';
+
+import { FormInput } from './FormInput';
+
+describe('FormInput', () => {
+  const defaultProps = {
+    label: 'Mail Id',
+    value: '',
+    onChange: vi.fn(),
+    'data-testid': 'form-input',
+  };
+
+  const FormInputWrapper = ({ value: initialValue = '', onChange, ...props }) => {
+    const [value, setValue] = useState(initialValue);
+    const handleChange = (e) => {
+      setValue(e.target.value);
+      if (onChange) onChange(e);
+    };
+    return <FormInput {...props} value={value} onChange={handleChange} />;
+  };
+
+  it('renderiza con props por defecto', () => {
```

Comment:
The test description strings are in Spanish (e.g. 'renderiza con props por defecto'). The project requires English as the standard for code, comments, and tests.

**Sugerencia:**
```typescript
Translate test descriptions to English. Example:

it('renders with default props', () => { ... });
```